restapi.go

```diff
@@ -2808,12 +2808,16 @@ func (s *Session) ApplicationCommand(appID, guildID, cmdID string) (cmd *Applica
 // ApplicationCommands retrieves all commands in application.
 // appID       : The application ID.
 // guildID     : Guild ID to retrieve all guild-specific application commands. If empty - retrieves global application commands.
-func (s *Session) ApplicationCommands(appID, guildID string) (cmd []*ApplicationCommand, err error) {
+func (s *Session) ApplicationCommands(appID, guildID string, withLocalizations bool) (cmd []*ApplicationCommand, err error) {
 	endpoint := EndpointApplicationGlobalCommands(appID)
 	if guildID != "" {
 		endpoint = EndpointApplicationGuildCommands(appID, guildID)
 	}
 
+	if withLocalizations {
+		endpoint += "?with_localizations=true"
+	}
```

Comment:
I totally overlooked the fact that the endpoint doesn't return full dicts without `with_localizations=true`.
However I think would be better to avoid the breaking change
```suggestion
    endpoint += "?with_localizations=true"
```